### PR TITLE
Add upload and help page

### PR DESCRIPTION
**Pull request by randombenj**

### CODE REVIEW

File: doc/getting-started.md [hunk 1]
```diff
@@ -1,21 +1,22 @@
-## Usage
+## Getting started with DOCAT
 
-### Add documentation
+### Uppload your documentation
```
Comment (fliiiix) [inline doc/getting-started.md]: typo

File: doc/getting-started.md [hunk 2]
```diff
@@ -1,21 +1,22 @@
...
 ```
 
 Any other file type is uploaded as well.
-An uploaded pdf would be available as
-`http://localhost:5000/api/awesome-project/1.0.0/my_awesome.pdf`
+An upploaded pdf could be viewed like this:
 
+`http://localhost/#/awesome-project/1.0.0/my_awesome.pdf`
 
+You can also manually upload your documentation.
```
Comment (fliiiix) [inline doc/getting-started.md]: yay

Comment (randombenj) [reply doc/getting-started.md]: :rofl:

**Comment by randombenj** [comment]
Will merge after adding the link to the help page

**Pull request merged**